networkgen/views.py

```diff
@@ -1,4 +1,9 @@
-from django.views.generic import TemplateView, View
+from django.views.generic import TemplateView, CreateView
+from django.urls import reverse
+from networkgen.forms import GeneratorForm
+from networkgen.models import Generator
+from django.views.generic.detail import DetailView
```

Comment:
Keep the imports sorted and tidy: first the python includes, then the thirdparty libraries sorted alphabetically, and lastly our code. Otherwise we are going to end up with double-imports or importing a whole package in individual chunks.